src/pages/Labels/labels.css

```diff
@@ -0,0 +1,8 @@
+.labels-section {
+    gap: var(--space-2);
+}
+
+.label-list-wrapper {
+    max-width: 500px;
```

Comment:
`rem` please

Reply:
I used pixels here because I've used rem majorly for text. And since I have used rem for text, I've also decreased the font-size percentage of the root in smaller devices to make text look readable and user-friendly across all devices. 
Hence, this caused the width of the sections/ containers to decrease, which I did not want. Hence the reason to stick with pixels!